src/TLuaInterpreter.cpp

```diff
@@ -12899,3 +12901,59 @@ Host& getHostFromLua(lua_State* L)
     assert(h);
     return *h;
 }
+
+int TLuaInterpreter::getColumnCount( lua_State * L )
+{
+    QString windowName;
+
+    if ( ! lua_gettop( L ) ) {
```

Comment:
The code formatting is not up to scratch yet - but that is understandable on a first contribution. 😄 

Assuming you have previously chosen the "Tools" -> "Options" -> "Beautifier" -> "ClangFormat" -> "Options" -> "Use Predefined Style" -> "File" option:
![screenshot_20171215_065826](https://user-images.githubusercontent.com/6163092/34030617-ab0a7144-e165-11e7-944c-029440c635a5.png)

You can using Qt Creator to select all the code you have edited and use "Tools" -> "Beautifier" -> "ClangFormat" -> "Format Selected Text":
![screenshot_20171215_065949](https://user-images.githubusercontent.com/6163092/34030588-98fdec06-e165-11e7-8f2f-cb3e898d90cd.png)
to fix up most of the formatting.

The only exception to this is that we use `{`...`}` for even single line `if`/`else` code blocks which some versions of clangFormat doesn't handle/fix auto-magically which you may need to fix further down this function.